tests/python/test_build_data.py

```
@@ -1848,6 +1843,188 @@ def fake_scrape_place(url: str, **_: object) -> SimpleNamespace:
+            "The area is equipped with toilets and is perfect for taking your children for a relaxing outing. "
+            "A wonderful place to unwind and enjoy nature. Highly recommended for families."
+        )
+
+        enrichment = build_data.normalize_place_page_enrichment(details)
+
+        self.assertEqual(enrichment.display_name, "Kumoba Pond")
+        self.assertIsNone(enrichment.description)
+
+        details.name = "Pink Street"
+        details.description = (
+            "Just an overrated place. Tiktok and instagram made it famous but I didn’t "
```

Ruff (0.15.12)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)